packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -695,6 +703,24 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
       debugBuildingDirtyElements = true;
       return true;
     }());
+
+    if (_needToReportFirstFrame && _reportFirstFrame) {
```

Comment:
Is there value in doing all of this in release mode? Or should this be guarded to only profile and debug mode?

Reply:
I added this to release mode so `_firstFrameCompleter` and thus `WidgetsBinding.firstFrameRasterized` is available in the release mode. It might be useful to handle the splash screen during add2app?